examples/stopwatch.rs

```diff
@@ -0,0 +1,81 @@
+// Draw some multi-colored geometry to the screen
```

Comment:
Forgot to update a comment? :P

Reply:
Oh man. I forgot so much stuff. I'll push a commit changing that and adding the changelog entry.